webpack.config.js

```diff
@@ -16,6 +16,20 @@ module.exports = {
 	...defaultConfig,
 	module: {
 		rules: [
+			{
+				test: /\.(png|jpe?g|gif)$/,
```

Comment:
@coreymcollins Looks good, but I think we should consider adding support for `.webp` and maybe `.svg`?

Reply:
Sure, I had initially not included SVG since we have those in their own icons directory but I could see a case where we have an SVG living outside of that directory.